spec/system/find/homepage/tracking_spec.rb

```diff
@@ -223,7 +223,7 @@ def and_all_accordions_are_open
 
   def when_i_browse_teacher_degree_apprenticeship_courses
     and_all_accordions_are_open
-    click_link_or_button "Browse teacher degree apprenticeship courses."
+    click_link_or_button "Browse all apprenticeship courses."
```

Comment:
Can we verify the assertions of this system spec to expect the apprenticeship courses?